core/src/main/scala/com/nvidia/spark/rapids/tool/profiling/ProfileArgs.scala

```diff
@@ -38,8 +38,11 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
         " rapids_4_spark_profile. It will overwrite any existing files" +
         " with the same name.",
       default = Some("."))
+  val driverlog: ScallopOption[String] =
+    opt[String](required = false,
+      descr = "Driver log filename - eg: /path/to/driverlog. Default is empty.")
   val eventlog: ScallopOption[List[String]] =
-    trailArg[List[String]](required = true,
+    trailArg[List[String]](required = false,
```

Comment:
this seems weird to me based on the usage above on line 31.  optional args are usually denoted with [ ].

Reply:
Eventlogs was required earlier but with this PR it becomes optional. If the user provides driver log then the tool will parse driver log and only generate unsupported operators.  I have updated the Usage section. Please let me know if it's confusing.